templates/publisher/listing.html

```diff
@@ -350,13 +382,37 @@
       ],
       'public_metrics_enabled': {{ public_metrics_enabled|tojson }},
       'public_metrics_blacklist': {{ join(public_metrics_blacklist, ',')|tojson }},
+      'license': {{ license|tojson }}
     },
     {% if error_list %}
       {{ error_list|tojson}}
     {% else %}
       null
     {% endif %}
     );
+
+    var licenseRadioContent = [
```

Comment:
Would be nice to have this licence init code somewhere in the module instead of inline script. Would it work inside of `market.initForm`? I guess screenshots are initialized in there.